components/lib/tooltip/Tooltip.js

```diff
@@ -14,6 +14,7 @@ export const Tooltip = React.memo(
         const [visibleState, setVisibleState] = React.useState(false);
         const [positionState, setPositionState] = React.useState(props.position || 'right');
         const [classNameState, setClassNameState] = React.useState('');
+        const [multipleFocusEvents, setMultipleFocusEvents] = React.useState(false);
```

Comment:
Actually you can switch this to a `ref` it doesn't need to be `useState` as its not used in any hooks you are just tracking a boolean state.

Reply:
I can update it but there is something I didn't understand help me if I'm missing something. I put boolean inside of state because of trigger a value with using `ref` how can I achieve that?

Reply:
you jiust do `yourRef.current = true` or `false`

Reply:
Yeap, I did the same but like I said we should trigger the component. You can try it in your local or I can commit.

Reply:
interesting i was not expecting that based on looking at this code.  You are just using a boolean flag.  Changing the state is not used by any hook?

Reply:
I think the issue is related about refs in React don't cause re-renders that's why when we use `state` instead of `ref` we can always work with changed and dynamic value.

Reply:
Ahhh OK so you need the re-render. Then I am OK how it is.

Reply:
Thanks!

Reply:
@melloware you merged but there is prettier error what should I do?

Reply:
Oh I can fix it. Next time run `npm run format` on the codebase it fixes it

Reply:
When will be the release date?